core/infra/protobuf/api.pb.cc

```diff
@@ -340,10 +340,11 @@ void protobuf_AssignDesc_api_2eproto() {
       GOOGLE_PROTOBUF_GENERATED_MESSAGE_FIELD_OFFSET(Transaction, _internal_metadata_),
       GOOGLE_PROTOBUF_GENERATED_MESSAGE_FIELD_OFFSET(Transaction, _is_default_instance_));
   ConsensusEvent_descriptor_ = file->message_type(14);
-  static const int ConsensusEvent_offsets_[3] = {
+  static const int ConsensusEvent_offsets_[4] = {
```

Comment:
Coders might want to read generated code to write code used protobuf (I want to see too), but following example generates *.pb.h and *.pb.cc file automatically:
https://github.com/google/protobuf/blob/master/examples/CMakeLists.txt#L43-L49